build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -83,7 +83,7 @@ public static BuildTargetResult RestoreTestAssetPackages(BuildTargetContext c)
             CleanNuGetTempCache();
 
             var dotnet = DotNetCli.Stage2;
-            dotnet.Restore("--verbosity", "verbose")
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
```

Comment:
Why do we --disable-parallel here and below?  That seems incorrect.  If there is a bug, we need to get that bug fixed ASAP.